### PR TITLE
feat : product page & sign-up

### PR DESCRIPTION
**Pull request by bhtibrewal**
Added Product page and signup

### CODE REVIEW

**Comment by yaswanthmyneni** [comment]
Add the cursor to the profile icon in navbar, give a margin to 'all the products' so cards will not be close to it. The code and the look of the page are good, can be merged.
Screenshot 2022-02-14 at 10 44 24 AM

File: style.css [hunk 1]
```diff
@@ -33,29 +33,125 @@ h3,
...
+}
+.items {
+  color: var(--primary-color);
+  font-size: 1.4rem;
+}
+
+/* subscribe modal */
+.subscribe-section-overlay,
+.login-section-overlay,
+.signup-section-overlay {
```
Comment (shangkaul) [inline style.css]: **Tip**: You can add a box shadow to darken the rest of the page when the modal is open. Try - ( box-shadow: 0 0 0 1600px rgba(0, 0, 0, 0.65);)
Screenshot 2022-02-14 at 12 00 54 PM

**Review by shangkaul — APPROVED** [review]
Great work on the UI! Looks good to merge🚀

File: pages/product/product.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+          <span class="checkbox-text">Abstraction</span>
+        </label>
+        <label><input type="checkbox" /> 
+          <span class="checkbox-text">Nature</span>
+        </label>
+        <label><input type="checkbox" /> 
+          <span class="checkbox-text">Landscape</span>
+        </label>
+      </div>
+      <div class="aside-sec">
```
Comment (Shivani-1524) [inline pages/product/product.html:66]: Great work on the UI but the hover on sort by could be made better. refer to these images:
image
image

File: pages/product/product.html [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+      <section class="products-sec">
+
+        <h2> All the Products</h2>
+        <div class="grid-3">
+          <div class="card w-30 text-o-media">
+            <div class="content">
+              <img
+                class="card-img"
+                src="https://render.fineartamerica.com/images/images-new-artwork/images-medium-5/la-porta-rossa-sulla-salita-guido-borelli.jpg?v=2"
+                alt=""
```
Comment (Shivani-1524) [inline pages/product/product.html:147]: make sure you've given appropriate text in all the alt tags for accessibility

**Review by Shivani-1524 — CHANGES_REQUESTED** [review]

**Review by thenorthkun — APPROVED** [review]
Apart from the changes suggested above by others, everything looks fine. Good to merge 👍

File: index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const loginSection = document.querySelector(".login-section-overlay");
+const signupSection = document.querySelector(".signup-section-overlay");
+
+const closeFunc = (e) => {
+    e.parentElement.style.display = "none";
+    console.log("clicked")
```
Comment (gauravsinhaweb) [inline index.js:7]: remove all console.log before commit.

File: index.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+}
+const openLogin = () => {
+    loginSection.style.display = "flex";
+    console.log("click")
+}
+const openSignup = () => {
+    loginSection.style.display = "none";
+    signupSection.style.display = "flex";
+    console.log("click")
```
Comment (gauravsinhaweb) [inline index.js]: remove all console.log before commit.

File: pages/homepage/homepage.html [hunk 6]
```diff
@@ -159,61 +151,99 @@ <h3>Art and Antiquity</h3>
...
+      <div class="flex-col signup-sec">
+        <i class="primary fa-regular fa-user fa-5x"></i>
+        <p class="body-l">Create my account on Artsy Shop!</p>
+
+        <div class="artsy-input">
+          <input type="text" />
+          <span class="input-label">Full Name</span>
+        </div>
+        <div class="artsy-input">
+          <input type="text" />
```
Comment (gauravsinhaweb) [inline pages/homepage/homepage.html]: use label with input for accessibility.

File: pages/homepage/homepage.html [hunk 7]
```diff
@@ -159,61 +151,99 @@ <h3>Art and Antiquity</h3>
...
+          <input type="password" />
+          <span class="input-label">Password</span>
+        </div>
+        <label class="flex-align-center">
+          <input type="checkbox" />
+          <span class="checkbox-text">
+            By registering, I accept the
+            <a class="primary" href="">General terms and conditions.</a>
+          </span>
+        </label>
```
Comment (gauravsinhaweb) [inline pages/homepage/homepage.html]: use an explicit label and give id to input in the form control.

File: pages/product/product.css [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+  background: white;
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-around;
+  padding: 2rem 5%;
+}
+.main-drop {
+  position: relative;
+  padding:1rem;
+  /* background: black; */
```
Comment (gauravsinhaweb) [inline pages/product/product.css]: remove unnecessary comments.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]
UI looks good. Good to merge after changes.

**Pull request merged**